### PR TITLE
New plan names

### PR DESCRIPTION
**Pull request by usirin**
/cc @sinan

### CODE REVIEW

File: client/home/lib/billing/components/subscriptioncontainer.coffee [hunk 1]
```diff
@@ -13,19 +13,15 @@ bongo = require 'app/redux/modules/bongo'
...
-  subscription.isTrial
-  subscription.trialDays
+  subscription.planAmount
   info.userCount
-  (isTrial, trialDays, userCount) ->
-    if isTrial
-    then trialTitles[trialDays]
-    else pluralize 'Developer', userCount, yes
+  (amount, userCount) ->
+    name = "$#{amount.dollars} Plan"
```
Comment (sinan) [inline client/home/lib/billing/components/subscriptioncontainer.coffee]:
```coffee
"$ #{if d = amount.dollars % 1 isnt 0 then d.toFixed(2) else d} Plan"
# 1 --> $1 Plan
# 9.9 --> $9.90 Plan
```

File: client/home/lib/billing/components/subscriptioncontainer.coffee [hunk 2]
```diff
@@ -13,19 +13,15 @@ bongo = require 'app/redux/modules/bongo'
...
+    type = if userCount is 1 then 'Solo' else userCount
+    pluralized = pluralize 'Developer', userCount
```
Comment (sinan) [inline client/home/lib/billing/components/subscriptioncontainer.coffee]:
```coffee
type = if userCount is 1 then 'Solo'
else if userCount <= 10 then 'Single Cloud'
else 'Dev Team'
```

**Review by sinan — CHANGES_REQUESTED** [review]
👍 minor changes needed

**Comment by usirin** [comment]
@sinan updated

**Review by sinan — APPROVED** [review]

**Pull request merged**